num_samples_per_second: 100